configs/angular-a11y.js

```diff
@@ -1,19 +1,21 @@
 module.exports = {
   plugins: ["@angular-eslint"],
+
   overrides: [
     {
       files: ["*.html"],
       plugins: ["@angular-eslint/template"],
       rules: {
+        "@angular-eslint/template/click-events-have-key-events": "warn",
+        "@angular-eslint/template/mouse-events-have-key-events": "warn",
+        "@angular-eslint/template/no-autofocus": "warn",
+
```

Comment:
Extra newline? Were these updated manually or through prettier or something?

Reply:
Not extra, on purpose. See other comment. =]